components/Layout.js

```diff
@@ -2,28 +2,90 @@ import Link from 'next/link';
 
 export const Layout = ({ children }) => (
   <>
-    <header className="w-full h-110 bg-stone-300">
-      <div className="w-full bg-light-50 p-2 md:p-5 grid grid-cols-4">
-        <Link href="/">
-          <a href="home">Hiswiki</a>
-        </Link>
+    <header className="w-full h-[440px] bg-[url('../styles/img/header_background.jpg')] bg-no-repeat bg-cover bg-stone-300">
+      <div className="w-full  bg-light-50">
+        <div className="max-w-screen-xl mx-auto p-2 md:p-5 flex">
+          <Link href="/">
+            <a
+              className="flex-initial font-yrsa font-bold uppercase text-4xl pr-8"
+              href="home"
+            >
+              Hiswiki
+            </a>
+          </Link>
+          <div class="flex-initial">
+            <form>
+              <label className="sr-only" for="search">
+                Vyhledávání
+              </label>
+              <input
```

Comment:
What is the difference between doing `aria-labelledby` and `<label for=...`

Reply:
from the mdn https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby
All interactive elements must have an accessible name. aria-labelledby can be used to reference another element to define its accessible name, when an element's accessible name needs to use content from elsewhere in the DOM.

Aria-labelledby takes precedence over all other methods of providing an accessible name, including aria-label, label, and the element's inner text.

Note that while using aria-labelledby is similar in this situation to using an HTML <label> element with the for attribute, there are some very important differences. The aria-labelledby attribute only defines the accessible name. It doesn't provide any of <label>'s other functionality, such as making clicking on the labeling element activate the input it is associated with. That has to be added back in with JavaScript.
========
It can be used, when you design your own dropdown select with your own style.